docs/literate/src/files/first_steps/changing_trixi.jl

```diff
@@ -0,0 +1,77 @@
+#src # Changing Trixi.jl itself
+
+# If you plan on editing Trixi.jl itself, you can download Trixi.jl locally and run it from
+# the cloned directory.
+
+
+# ## Cloning Trixi.jl
+
+
+# ### Windows
+
+# If you are using Windows, you can clone Trixi.jl by using the GitHub Desktop tool:
+# - If you do not have a GitHub account yet, create it on
+#   the [GitHub website](https://github.com/join).
+# - Download and install [GitHub Desktop](https://desktop.github.com/) and then log in to
+#   your account.
+# - Open GitHub Desktop, press `Ctrl+Shift+O`.
+# - In the opened window, paste `trixi-framework/Trixi.jl` and choose the path to the folder where
+#   you want to save Trixi.jl. Then click `Clone` and Trixi.jl will be cloned to your computer. 
+
+# Now you cloned Trixi.jl and only need to tell Julia to use the local clone as the package sources:
+# - Open a terminal using `Win+R` and `cmd`. Navigate to the folder with the cloned Trixi.jl using `cd`.
+# - Create a new directory `run`, enter it, and start Julia with the `--project=.` flag:
+#   ```shell
+#   mkdir run 
+#   cd run
+#   julia --project=.
+#   ```
+# - Now run the following commands to install all relevant packages:
+#   ```julia
+#   using Pkg; Pkg.develop(PackageSpec(path="..")) # Tell Julia to use the local Trixi.jl clone
+#   Pkg.add(["OrdinaryDiffEq", "Plots"])  # Install additional packages
+#   ```
+
+# Now you already installed Trixi.jl from your local clone. Note that if you installed Trixi.jl
+# this way, you always have to start Julia with the `--project` flag set to your `run` directory,
+# e.g.,
+# ```shell
+# julia --project=.
+# ```
+# if already inside the `run` directory.
+
+
+# ### Linux
+
+# You can clone Trixi.jl to your computer by executing the following commands:
+# ```shell
+# git clone git@github.com:trixi-framework/Trixi.jl.git 
+# # If an error occurs, try the following:
+# # git clone https://github.com/trixi-framework/Trixi.jl
+# cd Trixi.jl
+# mkdir run 
+# cd run
+# julia --project=. -e 'using Pkg; Pkg.develop(PackageSpec(path=".."))' # Tell Julia to use the local Trixi.jl clone
+# julia --project=. -e 'using Pkg; Pkg.add(["OrdinaryDiffEq", "Plots"])' # Install additional packages
+# ```
```

Comment:
As I encountered an error here recently, we should add:

If you encounter errors when executing `julia --project=. -e 'using Pkg; Pkg.develop(PackageSpec(path=".."))'` try 
`julia --project=. -e using Pkg; Pkg.update()`

Reply:
Sounds good, but should probably be `julia --project=. -e 'using Pkg; Pkg.update()'`. Note the ticks.

Reply:
Are you sure that this is more helpful rather than confusing for the intended target audience, i.e., Julia novices? I've never encountered this error when doing a new setup, thus I'd just omit it.

Reply:
I will make a seperate PR for this.

Reply:
See #1831